setup.py

```diff
@@ -26,5 +33,6 @@
     ],
     install_requires=install_requires,
     packages=setuptools.find_packages(),
-    include_package_data=True
+    include_package_data=True,
+    entry_points = { 'console_scripts': ['irida_staramr_results=irida_staramr_results.cli:main']}
```

Comment:
We should use the command `irida-startamr-results` instead of `console_scripts`, since this command will be how our users will execute the program, so the name should reflect it.

Reply:
Whoops I think I didn't have enough coffee before reviewing

Reply:
Done. But I'm having trouble with modules when I run the command. Getting the ModuleNotFoundError